activerecord/test/cases/adapters/mysql/schema_test.rb

```diff
@@ -67,11 +67,11 @@ def test_primary_key
 
```

Comment:
This 2 test cases validate how AR behaves when `table_name` has shema name, we can change these to `data_sources`.